myapp/config/locales/common.ja.yml

```
@@ -0,0 +1,7 @@
+ja:
+  common:
+    edit: "編集"
+    delete: "削除"
+    top_page: "トップページ"
+    messages:
+      confirm_delete: "本当に削除しますか？"
```

【FYI】

It's just my personal opinion, but you should not use "common". 
Anything that is not a good decision tends to be put in here...


```suggestion
  action:
    edit: "編集"
    delete: "削除"
  page_title:
    top_page: "トップページ"
  messages:
    confirm_delete: "本当に削除しますか？"
```

---

Since `action` is a name clash with Rails' action, it might be better to use `ops` or something like that.

---

Thank you for suggestion.
I'll modify it.